src/platform/utilities/sso/index.js

```diff
@@ -27,15 +27,22 @@ export async function ssoKeepAliveSession() {
   }
 }
 
-export async function checkAutoSession() {
+export async function checkAutoSession(application = null, to = null) {
   await ssoKeepAliveSession();
   if (hasSession() && hasSessionSSO() === false) {
     // explicitly check to see if the SSOe session is false, as it could also
     // be null if we failed to get a response from the SSOe server, in which
     // case we don't want to logout the user because we don't know
     logout('v1', 'sso-automatic-logout');
   } else if (!hasSession() && hasSessionSSO() && !getForceAuth()) {
-    login('idme', 'v1', null, null, { inbound: 'true' }, 'sso-automatic-login');
+    login(
+      'idme',
+      'v1',
+      application,
```

Comment:
@hinzed1127 when not in use, these parameters will now be `undefined` and not `null` right?  If so does the `login` function still behave in the same way?

Reply:
The default values in `checkAutoSession` and the `AutoSSO` props are both set to `null`, so it should behave the same as before when that was explicitly passed values to `login`